code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -0,0 +1,88 @@
+#define CALLING_BURROWED_DURATION 15 SECONDS
+
+// ***************************************
+// *********** Call of the Burrowed
+// ***************************************
+/datum/action/xeno_action/call_of_the_burrowed
+	name = "Call of the Burrowed"
+	action_icon_state = "lay_egg"
+	cooldown_timer = 5 MINUTES
+	keybind_signal = COMSIG_XENOABILITY_CALL_OF_THE_BURROWED
+
+
+/datum/action/xeno_action/xeno_action/call_of_the_burrowed/action_activate()
+	var/mob/living/carbon/xenomorph/shrike/S = owner
+	S.calling_larvas = TRUE
+
+	notify_ghosts("\The <b>[S]</b> is calling for the burrowed larvas to wake up!", source = S, NOTIFY_JOIN_AS_LARVA)
+
+	addtimer(CALLBACK(S, /mob/living/carbon/xenomorph/shrike.proc/calling_larvas_end), CALLING_BURROWED_DURATION)

```

Comment:
Do you need the full path there like this?

Isn't passing in the `S` enough?
```suggestion
	addtimer(CALLBACK(S, .proc/calling_larvas_end), CALLING_BURROWED_DURATION)
```

Reply:
It is necessary, src here is the /datum/action and you want to call it on the xeno mob type.